### PR TITLE
more readme updates and pg tasks

### PR DESCRIPTION
**Pull request by jchappelow**
The commands all run but I didn't carefully proofread this since I want it reviewable asap and it may conflict with https://github.com/kwilteam/kwil-db/pull/549

[Rendered.](https://github.com/kwilteam/kwil-db/blob/d4a61ad588524d303d2ddee2b0fbc8b626b94926/README.md)

Complements https://github.com/kwilteam/docs/pull/82

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,33 +12,98 @@ Learn more about Kwil at [kwil.com](https://kwil.com).
...
+
+`task pg` may be used to run the above command.
 
-You can start a single node network using the `kwild` binary built in the step above:
+You can then start a single node network using the `kwild` binary built in the previous section:
 
 ```shell
-.build/kwild --autogen
+# Use the full path to kwild if it is not on your PATH.
+kwild --autogen
```
[the remark this answers is not in the document]
Comment (jchappelow) [reply README.md:74]: Was changed to 127.0.0.1 default a few days ago in https://github.com/kwilteam/kwil-db/commit/147895a48e489ce94abe7784919e36dbd7075e75
Sure you're on latest main?

[the remark this answers is not in the document]
Comment (jchappelow) [reply README.md:74]: Oh, you probably have a `~/.kwild/config.toml` already from a previous run

[the remark this answers is not in the document]
Comment (brennanjl) [reply README.md:74]: Ah I had not pulled main. Works now

**Review by brennanjl — APPROVED** [review]

**Comment by jchappelow** [comment]
conflicts resolved, but I feel weird about the section headings.
Also Docker isn't is build prerequisite, so it's strange to have it in that list.  If we want a list of runtime requirements, it would go there, but it's the only one so there's really no such list.

**Comment by jchappelow** [comment]
Latest renderd: https://github.com/kwilteam/kwil-db/blob/d4a61ad588524d303d2ddee2b0fbc8b626b94926/README.md

**Pull request merged**